SourceKitten.xcworkspace/xcshareddata/SourceKitten.xcscmblueprint

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Generally, I think Xcode putting something in `xcshareddata` is Apple's way of recommending that it be shared amongst users of the project, and therefore checked in to source control.